spec/Subscriber-spec.ts

```diff
@@ -262,7 +262,7 @@ describe('Subscriber', () => {
       const subscription = of(42).subscribe(observer);
 
       observer = undefined;
-      global.gc();
+      global.gc?.();
```

Comment:
The `?.` is necessary despite the inclusion of `global.gc` in the `if` statement's expression, as the invocation is made within a callback, etc.